src/Hooks.cpp

```diff
@@ -356,7 +356,7 @@ namespace NLA
 		void Install() {
 			const REL::Relocation<std::uintptr_t> combatControllerUpdate{ RELOCATION_ID(43162, 44384) };
 			const REL::Relocation<std::uintptr_t> weaponFire{ RELOCATION_ID(17693, 18102) };
-			const REL::Relocation<std::uintptr_t> launchSpell{ RELOCATION_ID(33672, 34452) };
+			const REL::Relocation<std::uintptr_t> launchProjectile{ RELOCATION_ID(42928, 44108) };
```

Comment:
This change was done due to remove the offset, and also this was actually already reversed for VR, so changed this when looking for matching VR ids

Reply:
Sorry, I didn't get that. This was supposed to hook one specific call to `LaunchProjectile` where NPCs launching a spell. But with this change is it now hooking all calls to `LaunchProjectile`?

Reply:
Oh.. ofc... haha my bad, then I guess it's 3 missing IDS for VR then :D

Reply:
I see, well, I had VR exe somewhere laying around. Thanks for setting it up! Do you know how could I pass like 3 IDs together in a single relocation?